content/documentation/zooming-images.md

```diff
@@ -0,0 +1,207 @@
+type: essay
+abstract: "Adding high-resolution zooming images to your project"
+---
+
+To add high-resoultion zooming images to your project, Quire uses the [International Image Interoperability Framework (IIIF)](https://iiif.io/). Within your Quire project, you can either point to existing IIIF assets, or create your own. IIIF zooming images can be displayed within your Markdown pages using the `q-figure-zoom` shortcode, or in the built-in image viewer on entry pages. (Learn more about entry pages in the [*Collection Catalogues*](/documentation/collection-catalogues) section of this documentation.)
+
+## What is IIIF?
+
+IIIF (referred to when speaking as "Triple I F"), stands for the International Image Interoperability Framework. It is a shared, open set of standards for storing images and image data that allows for seamless and regularized sharing and display of those images across different uses within an institution, or across different institutions. While IIIF images offer a wide variety of features and applications, Quire uses them specifically for deep zooming.
+
+{{< q-figure-zoom id="irises" >}}
+
+You can either point Quire to an existing IIIF repository of images, or add high-resolution images into a Quire project directly. When adding your own images, Quire processes them into hundreds of individual image tiles. Each tile comprises a portion of the image at a particular zoom level to then be displayed in our existing zooming image viewer. Because the images have been tiled, users of your Quire project only need their browsers to load the tiles of the image portion and zoom level they're looking at at the moment. This means much faster page loads, and higher-resolution images.
+
+{{<q-figure id="iris-zoom" >}}
+
+## Use Existing IIIF Images
+
+To include existing IIIF images in your project, you will need the URL for the `info.json` file for that image, as well as a static fallback version of the image (typically a JPG) for use in Quire’s PDF and e-book outputs.
+
+The `info.json` file is [a required component of all IIIF images](https://iiif.io/api/image/3.0/#5-image-information) and includes basic information about the full image and the availability of image tiles at different zoom levels.
+
+If you are working with your own institution’s IIIF repository, your digital department will be able to help you identify the `info.json` URLs you need. If you are working with other, open IIIF repositories, you may need to do some digging to find the right URLs. Oftentimes, institutions will provide public access to the IIIF `manifest.json` file for a given object in their collection. This is a more fullsome IIIF document that includes much more information about the image and the resources available. Within that `manifest.json` file however, you can look under `sequences[0].canvases[0].images[0].resource.service.@id`. The URL in the `@id` field, when appended with `/info.json`, will be what you need to add the image to your Quire project.
```

Comment:
It's unclear to me what you mean when you say "go digging" to find the `manifest.json`. Does that mean they navigate to a section of the website or reach out to someone specific? I'm sure every institution is different but if I read this, I wouldn't know where to begin.

Reply:
Good point. I'll add a few specific examples.